jenkins/vs/buildimage-vs-image-201911/Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 pipeline {
-    agent { node { label 'jenkins-kvm-workers' } }
+    agent { node { label 'jenkins-build-workers' } }
```

Comment:
we need to separate this job like what we did for builimage-vs-image before we change the tag.

Reply:
Where are we separating the job? There's no "Test" stage for any of these three jobs, so I'm not sure where the split should be,